lisa/tools/modinfo.py

```diff
@@ -59,7 +59,11 @@ def get_version(
             no_info_log=True,
             no_error_log=True,
         )
-        return get_matched_str(output, self._version_pattern)
+        for line in output.splitlines():
+            version = get_matched_str(line, self._version_pattern)
+            if len(version) > 0:
+                return version
+        return ""
```

Comment:
If no version is found, should it throw an error? or is this behaviour normal for some distros?

Reply:
some modules don't have version